### PR TITLE
Get profile data from Mozillians.org

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

**Comment by johngian** [comment]
Fixes #28

File: woodstock/voting/utils.py [hunk 1]
```diff
@@ -8,44 +8,58 @@
...
 
-        url = MOZILLIANS_API_URL + '?' + urllib.urlencode(data)
-        resp = requests.get(url)
+    if not queryset:
+        queryset = MozillianProfile.objects.all()
 
-        if not resp.status_code == 200:
-            logger.error('Reponse status: {}'.format(resp.status_code))
+    for mozillian in queryset:
+        data = get_mozillian_by_username(mozillian.username)
```
Comment (akatsoulas) [inline woodstock/voting/utils.py]: s/username/mozillian_username

**Comment by akatsoulas** [comment]
r+wc

**Comment by akatsoulas** [comment]
Also please remove the last commit. Not needed any more.

**Pull request merged**